packages/features/bookings/Booker/Booker.tsx

```diff
@@ -140,7 +144,16 @@ const BookerComponent = ({
       scrolledToTimeslotsOnce.current = true;
     }
   };
-
+  useEffect(() => {
+    if (connectedCalendars.length > 0 && hasSession) {
```

Comment:
'connectedCalendars' are passed by the 'calendars' prop, which is passed to 'Booker' by the 'BookerWebWrapper', which gets the 'calendars' using 'useCalendars' hook.

However, if we check 'useCalendars' hook, then fetching connected calendars is enabled only if the setting is set in state or 'overlayCalendar' exists as search param:
```
const { data, isPending } = trpc.viewer.connectedCalendars.useQuery(undefined, {
    enabled: !!calendarSettingsOverlay || Boolean(searchParams?.get("overlayCalendar")),
  });
```

But the newly added `useEffect` only sets 'overlayCalendar' to true, if there are connected calendars, so we have a closed circle - I tried testing this PR and even though I have connected a calendar visiting someone's booking page did nothing.

Reply:
Yeah, that's true. I will fix that